CSharp/Samples/Integral/README.md

```diff
@@ -19,18 +19,18 @@ You can obtain these values by publishing the Knowledge base, using *Publish* bu
 
 ![Knowledge Base Published](images/qnamaker-publish.png)
 
-Now update the `QnAMaker_SubscriptionId` and `QnAMaker_KnowledgeBaseId` appSettings in the [Web.config](Web.config#L18-L19).
+Now update the `QnAMakerSubscriptionKey` and `QnAMakerKnowledgeBaseId` appSettings in the [Web.config](Web.config#L18-L19).
 
 ### LUIS Application
 If you want to test this sample, you have to import the pre-build [LUIS_MODEL.json](../LuisActions/LUIS_MODEL.json) file to your [LUIS account](https://luis.ai/).
 
-The first step to using LUIS is to create or import an application. Go to the home page, www.luis.ai, and log in. After creating your LUIS account you'll be able to Import an Existing Application where can you can select a local copy of the LUIS_MODEL.json file an import it.
+The first step to using LUIS is to create or import an application. Go to the home page, www.luis.ai, and log in. After creating your LUIS account you'll be able to Import an Existing Application where can you can select a local copy of the LUIS_MODEL.json file and import it.
 
 ![Import an Existing Application](../LuisActions/images/prereqs-import.png)
 
 Once you imported the application you'll need to "train" the model ([Training](https://www.microsoft.com/cognitive-services/en-us/LUIS-api/documentation/Train-Test)) before you can "Publish" the model in an HTTP endpoint. For more information, take a look at [Publishing a Model](https://www.microsoft.com/cognitive-services/en-us/LUIS-api/documentation/PublishApp).
```

Comment:
This link is broken. Doesn't go anywhere.

Reply:
The publishing link also appears to be broken.

Reply:
Fixed broken links